config.py

```diff
@@ -4,6 +4,8 @@
 
 class Config:
     SECRET_KEY = os.getenv('SECRET_KEY') or 'hardtoguessstring'
+    REDIS_DATABASE_URI = os.getenv('DEV_REDIS_URL')
+    REDIS_DATABASE_URI = os.getenv('PROD_REDIS_URL')
```

Comment:
@michael-basweti  why are the keys the same?

Reply:
Those were set for testing and they have been removed